### PR TITLE
Change the permissions of the binaries at build time so they are executable

### PR DESCRIPTION
**Pull request by yanfali**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

### CODE REVIEW

File: osx/qmk_toolbox/Info.plist [hunk 1]
```diff
@@ -108,7 +108,7 @@
 		</dict>
 	</dict>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright 2017 Jack Humbert</string>
+	<string>Copyright 2019 Jack Humbert</string>
```
Comment (fauxpark) [inline osx/qmk_toolbox/Info.plist]: https://github.com/qmk/qmk_toolbox/pull/78#discussion_r287890851

File: osx/QMK Toolbox.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -256,13 +296,23 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```
Comment (fauxpark) [inline osx/QMK Toolbox.xcodeproj/project.pbxproj]: All of this noise should be in a separate PR for updating the Xcode project.

File: osx/QMK Toolbox.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -218,6 +221,43 @@
...
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\nset -e\nchmod 666 ./dfu-util ./avrdude ./teensy_loader_cli ./dfu-programmer\n";
```
Comment (fauxpark) [inline osx/QMK Toolbox.xcodeproj/project.pbxproj]: If the flashers were moved into a dedicated directory this could look better.

File: osx/QMK Toolbox.xcodeproj/project.pbxproj (hunk 2 above)
Comment (yanfali) [reply osx/QMK Toolbox.xcodeproj/project.pbxproj]: Agrrerd.

File: osx/QMK Toolbox.xcodeproj/project.pbxproj (hunk 3 above)
Comment (yanfali) [reply osx/QMK Toolbox.xcodeproj/project.pbxproj]: Great idea!

**Comment by yanfali** [comment]
@fauxpark as outlined on discord, this PR is for demonstration purposes only. But I like where you are going with this.

File: osx/QMK Toolbox.xcodeproj/project.pbxproj (hunk 3 above)
Comment (yanfali) [reply osx/QMK Toolbox.xcodeproj/project.pbxproj]: How about we move all the executables to bin/ ?

Comment (fauxpark) [reply osx/QMK Toolbox.xcodeproj/project.pbxproj]: Sounds good to me.
